src/shim/sdk/tests/DocumentSDK.test.ts

```diff
@@ -319,6 +293,9 @@ describe("DocumentSDK", () => {
                         },
                         [document]
                     );
+                    //Ensure we cloned these bytes before passing them to the frame
+                    const passedDoc = (FrameMediator.sendMessage as jasmine.Spy).calls.mostRecent().args[0].message.documentData;
+                    expect(passedDoc).not.toBe(document);
```

Comment:
Same here and the rest of the spots this is being checked, unless it's covered somewhere lower down/more specific.

Reply:
Ditto from above, asserted about in the `.toHaveBeenCalledWith` assertion.